rmcreyes/cpen391-backend
# `isConfirmed` meter status admin update

# :ship: Pull Request

## :question: Purpose

Bug: user confirmed, and then admin checks meter status but meter still shows not confirmed

## :hammer: Validation Strategy

- Park as user or guest
- Check meter status (`isConfirmed: false` expected since not yet confirmed)
- User confirms license plate (`isNew: true` and/or `isNew: false`)
- Check meter status (`isConfirm: true` should be expected)
